src/engine/renderer/tr_image_jpg.cpp

```diff
@@ -102,7 +102,7 @@ void LoadJPG( const char *filename, unsigned char **pic, int *width, int *height
 
 	byte *buf;
 #if JPEG_LIB_VERSION < 80
```

Comment:
Missed some spots...

Reply:
My bad, forgot to push the latest version.